civis/ml/_model.py

```diff
@@ -819,6 +820,13 @@ def train(self, df=None, csv_path=None, table_name=None,
         polling_interval : float, optional
             Check for job completion every this number of seconds.
             Do not set if using the notifications endpoint.
+        validate : bool, optional
```

Comment:
We'd like to be able to support user-provided test sets in the future, and I think it would be better to be able to use one parameter to switch between the default of cross-validating over the training data, not testing the model, and testing on user-provided data. Do you think this parameter will still be usable when the extra input options are available?

Reply:
Maybe it would be better to have a `validate_data` parameter instead, which defaults to `'train'`, but could be `None` for no validation, or a file_id if the user provides a separate validation set?

Reply:
Yes, that could work. Maybe `validation_data` or `test_data`? `validate_data` sounds more like it's asking about data quality checks on the training data.